report/renderer/dom.js

```diff
@@ -154,6 +154,9 @@ export class DOM {
    * @param {string} url
    */
   safelySetHref(elem, url) {
+    // Defaults to '' to fix proto roundtrip issue.
+    url = url || '';
```

Comment:
doing this instead of default param value, because that makes typescript make the second param optional.